ssg/build_ovals.py

```diff
@@ -267,13 +271,20 @@ def checks(env_yaml, yaml_path, oval_version, oval_dirs):
     for _dir_path in find_rule_dirs(guide_dir):
         rule_id = get_rule_dir_id(_dir_path)
 
+        rule_path = os.path.join(_dir_path, "rule.yml")
+        rule = Rule.from_yaml(rule_path, env_yaml)
+
+        local_env_yaml['rule_id'] = rule.id_
+        local_env_yaml['rule_title'] = rule.title
+        local_env_yaml['rule_description'] = re.sub('<.*?>', '"', rule.description)
```

Comment:
Don't forget to remove `rule_description` here.

Reply:
Removed.